src/components/FormPageComponents/AppBar.js

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import logo from "../../images/logo.png";
+import { makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles({
+  imgStyle: {
+    width: "150px",
```

Comment:
Dont hardcode button size, will looks different on 4k and on small screen. Why you don't put those butttons inside grid items

Reply:
![image](https://user-images.githubusercontent.com/108327339/177986043-d4ee3fe9-2774-4200-82f3-c57c43619343.png)
?